src/js/components/modals/ServiceFormModal.js

```diff
@@ -209,6 +210,7 @@ class ServiceFormModal extends mixin(StoreMixin) {
 
   onMarathonStoreServiceEditError(errorMessage) {
     this.setState({
+      force: errorMessage.message && /force=true/.test(errorMessage.message),
```

Comment:
What do you think about putting this `/force=true/.test...` check in a separate method on the `ServiceFormModal`? Something like `#shouldForceEdit` that accepts the `message` and returns `/force=true/.test(message)`.